sdk/data/azappconfig/response_types.go

```diff
@@ -78,3 +79,70 @@ type SetSettingResponse struct {
 	// SyncToken contains the value returned in the Sync-Token header.
 	SyncToken SyncToken
 }
+type ArchiveSnapshotResponse struct {
+	generated.Snapshot
+
+	// Link contains the information returned from the Link header response.
+	Link *string
+
+	// SyncToken contains the information returned from the Sync-Token header response.
+	SyncToken *string
```

Comment:
Since a response always contains a sync token, these don't need to be pointer-to-type.